packages/psss/src/__tests__/period.test.js

```diff
@@ -36,25 +38,23 @@ describe('period', () => {
   });
 
   it('Gets Days Till Due Day', async () => {
-    const sunday = new Date(1606561200000); // Sun 29th November 2020
-    const monday = new Date(1606647600000); // Mon 30th November 2020
-    const tuesday = new Date(1606734000000); // Tuesday 1st December 2020
-    const wednesday = new Date(1606820400000); // Wednesday 2nd December 2020
+    const sunday = '2020-11-29';
```

Comment:
Tests would pass in NZ but not in Melbourne

Reply:
Great catch!